src/main/java/org/skriptlang/skript/lang/entry/EntryValidator.java

```diff
@@ -87,6 +85,12 @@ public EntryContainer validate(SectionNode sectionNode) {
 			while (iterator.hasNext()) {
 				EntryData<?> data = iterator.next();
 				if (data.canCreateWith(node)) { // Determine if it's a match
+					if (data instanceof SubContainerEntryData subContainer) {
+						if (!subContainer.validate((SectionNode) node)) {
+							ok = false;
```

Comment:
I don't know if it's appropriate to set this to false here. If you have two datas that both match a node, but one fully matches the subcontainer and the other doesn't, then ok will be false but we did find a successful match.